.rubocop.yml

```diff
@@ -1,46 +1,33 @@
-inherit_from: .rubocop_todo.yml
+require:
+  - rubocop-minitest
+  - rubocop-performance
+  - rubocop-rake
 
 AllCops:
-  TargetRubyVersion: 2.5
+  NewCops: enable
+  TargetRubyVersion: '2.5'
+  Exclude:
+    - 'test/**/*'
```

Comment:
I don't think we should disable RuboCop for all tests. It can be very useful.

Reply:
@ekohl agree! i've updated changes which fix some of the cops.